### PR TITLE
Feature/linters

### PR DESCRIPTION
**Pull request by Markotweyo**
I added Hello Microverse Project and installed linters and set the stylelintrc.json file to fix CSS lint errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# hellomicroverse
+# Project Name
+
+> Hello World Microverse Project.
+
+
```
Comment (danmainah) [inline README.md:6]: - [ ] Kindly add more description about your project, this is usually the first point of interaction between your project and the world, showcasing what the project is all about and how to use it..

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Markotweyo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️

- [ ] Kindly make sure to add pull request summary, explaining in details changes introduced to the pull request.

_Check the comments under the review._

## Optional suggestions

- [ ] Kindly make sure to add more content to your readme file,in its current state it has very little information.
- [ ] In future projects make sure to use correct tense in your commit messages.For example instead of `Added test.md file` you san `Add test.md file`.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (Markotweyo) [reply README.md:6]: Thank you for the review. I have added the pull request comments

**Review by danmainah — APPROVED** [review]
Hi @Markotweyo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description

## Optional suggestions

- [ ]  Kindly make sure to add more content to your readme file,in its current state it has very little information.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**